src/App.jsx

```diff
@@ -1,8 +1,17 @@
 const App = () => {
   return (
+<<<<<<< HEAD
```

Comment:
This is becuse of merge conflict.

Change this entire code to return only this 

```
<h1 className="text-orange bg-brown text-4xl font-bold">
    Hello World!
</h1>
```

Reply:
I have done this